### ISSUE TITLE
Расширение VSCode для работы с ЦСЯ Markdown

### ISSUE THREAD

**Issue by 0xGeorgii** [slavonic/tooling-todo#2]
1. [x] Рендеринг в HTML в реальном времени
2. [x] Конвертирование в
      - [ ] XML
      - [ ] HTML
      - [x] LaTeX

**Comment by 0xGeorgii** [comment]
@typiconman, @pgmmpk please add more info whether it should be based on existing project or a new one. also, it will be great if you have already had a vision of how it should look like and share it. Or I will research my own 🎡  😸

**Comment by pgmmpk** [comment]
@GeorgePlotnikov 
Начать предлагаю вот тут:

https://github.com/slavonic/vscode-church-slavonic

Рендеринг в HTML уже работает. Хочется добавить команды для конвертации в XML и LaTeX.

В markdown-it-church-slavonic есть эта функциональность (в зачаточном состоянии).

**Comment by typiconman** [comment]
ЦСЯ в LaTeX см. тут: https://github.com/slavonic/cu-tex

**Comment by 0xGeorgii** [comment]
Am I right you want to add commands for the command palette (Ctrl+Shift+P) for dumping current church Slavonic to the LaTeX and XML [as a file in at the same directory]? something like that?

**Comment by pgmmpk** [comment]
Да, хочется команду, в палитре. Например "csl-generate-xml", которая создаст новый файл и откроет его в новой табе редактора.

**Comment by 0xGeorgii** [comment]
Can I close it? I think the particular implementation of rendering is up to the MD plugin itself.

**Comment by pgmmpk** [comment]
lets keep this open till we can generate a satisfactory latex and xml

**Comment by 0xGeorgii** [comment]
ok.

[PR 1](https://github.com/slavonic/vscode-church-slavonic/pull/2)
[PR 2](https://github.com/slavonic/markdown-it-church-slavonic/pull/5)
